lib/galaxy/security/__init__.py

```diff
@@ -776,6 +776,34 @@ def get_private_user_role(self, user, auto_create=False):
                 return None
         return role
 
+    def get_admin_role(self, name):
```

Comment:
Maybe `def get_role(self, name, type=self.model.Role.types.ADMIN):` and similar below for `create_admin_role()`, so these are more usable in the general case.